static/library/map_component.ts

```diff
@@ -245,7 +256,10 @@ export class DatacommonsMapComponent extends LitElement {
       },
       showExploreMore: this.showExploreMore,
       statVarSpec: {
-        denom: "",
+        denom:
```

Comment:
looks like this is the empty variable case, so maybe keep denom as `""`

Reply:
Good call